dapps/assets/src/components/contacts/Contacts.vue

```diff
@@ -78,7 +78,7 @@
           :fields="columns"
           :filter="filter"
           :filter-included-fields="filterBy"
-          :show-empty="true"
+          :show-empty="!isLoading"
```

Comment:
Does this solve the flickering? If so, cool! :)

Reply:
Yes! It only shows the is empty, when loading has finished :)